### DISCUSSION TITLE
How about adding Hertzbeat Website monitoring to Matomo

### DISCUSSION THREAD

**Discussion by zhangshenghang** [apache/hertzbeat#2764] [Ideas]
How about adding Hertzbeat Website monitoring to Matomo? It can analyze the page statistics of our users visiting the website, which can help us analyze which monitoring users use more frequently.

currently, many projects have been supported
<img width="444" alt="image" src="https://github.com/user-attachments/assets/e4240074-00f2-4aec-8439-08badbe54756">

e.g.
https://analytics.apache.org/index.php?module=CoreHome&action=index&date=today&period=day&idSite=1&updated=1#?period=day&date=today&idSite=1&category=Dashboard_Dashboard&subcategory=1

<img width="1438" alt="image" src="https://github.com/user-attachments/assets/2fe98dbe-1412-4a5d-8e6c-c728cd7e42c5">

**Comment by tomsun28** [comment]
+1 good idea.
